.vscode/launch.json

```diff
@@ -96,10 +96,14 @@
       "port": 9091,
       "cwd": "${workspaceFolder}",
       "envFile": "${workspaceFolder}/.flaskenv",
-      "env": { "PYTHONPATH":"${workspaceFolder}"},
+      "env": {
+        "PYTHONPATH":"${workspaceFolder}",
+        "DATABASE_URL":"postgresql://postgres:password@localhost:5432/assessment_store",
+        "APPLICATION_STORE_API_HOST": "http://localhost:3002"
+      },
       "justMyCode": false,
-      "args": ["--roundid", "c603d114-5364-4474-a0c4-c41cbf4d3bbd",
-        "--app_type", "COF"]
+      "args": ["--roundid", "fc7aa604-989e-4364-98a7-d1234271435a",
+        "--app_type", "NSTF"]

```

Comment:
As above - can you add to the readme how to configure for different funds/rounds?